ext/openssl/ossl_ssl.c

```diff
@@ -1521,6 +1521,9 @@ ossl_start_ssl(VALUE self, int (*func)(), const char *funcname, VALUE opts)
     int ret, ret2;
     VALUE cb_state;
     int nonblock = opts != Qfalse;
+#if defined(SSL_R_CERTIFICATE_VERIFY_FAILED)
```

Comment:
This #if may be unnecessary, but I'm not sure if these error codes are meant to be part of the public API.